Anabaena/index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      language="javascript"
+      type="text/javascript"
+      src="../libraries/p5.min.js"
+    ></script>
+    <script type="module">
+      import { sketch } from "./Anabaena.js";
+      import "../sketchlib/sketchbook-navbar.js";
+      const p5_sketch = new p5(sketch, "sketch");
+    </script>
+    <link rel="stylesheet" href="../styles/layout.css" />
+    <script
+      id="MathJax-script"
+      async
+      src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"
+    ></script>
+  </head>
+  <body>
+    <sketchbook-navbar></sketchbook-navbar>
+    <div class="container">
+      <h1>Anabaena Catenula (2025)</h1>
+      <div id="sketch"></div>
+      <div class="description">
+        <p>
+          Move the mouse (desktop) or touch and drag the canvas (mobile) to move
+          the bacterium around.
+        </p>
+        <p>
+          This sketch is a combination of the animation technique used in the
+          <a href="../Worm/">Worm</a> sketch, but also uses an L-system to
+          simulate bacterial growth.
+        </p>
+        <p>
+          The L-system I'm using is based on the L-system described in
+          <cite
+            >Section 1.2 of
```

Comment:
Now that I actually have this in front of me, I realize my simulation isn't quite right. I was focusing too much on the _gerbera_ paper on the same website. The L-system is essentially the same (oriented Fibonacci growth), however the timing of growth/division works a little differently when looking at figure 6.3

- The change in length should happen in the S cells, not the L cells! Essentially the cell starts with length 1 (just born) grows to 2 (adult) and further to 3 (ready to divide)
- The book starts with a mature L cell rather than an S cell.
- Also note that chapter 6 talks about exponential growth being smoother than linear growth

Reply:
That said... I don't yet have transformations added to my primitives system.

🤔Maybe just use line segments and set the stroke width thicker for now?